browser/mapping_test.go

```diff
@@ -511,6 +511,81 @@ type responseAPI interface {
 	URL() string
 }
 
+// Strict mode:
+// All operations on locators throw an exception if more
+// than one element matches the locator's selector.
+//
+// See Issue #100 for more details.
```

Comment:
As a general comment, to me it seems odd that we have moved all these comments for all interfaces to a test file. Would it make more sense to move them to the type implementation instead?

Reply:
Makes sense 👍

Reply:
See: af05e07